include/SDL3/SDL_hints.h

```diff
@@ -1760,6 +1760,14 @@ extern "C" {
  */
 #define SDL_HINT_JOYSTICK_HIDAPI_SINPUT "SDL_JOYSTICK_HIDAPI_SINPUT"
 
+ * This variable can be set to the following values:
```

Comment:
You missed the opening `/**` before this line, that's why compilers are screaming.